scripts/build/build/targets.py

```diff
@@ -619,6 +619,7 @@ def Bl602Targets():
     target = Target('bl602', Bl602Builder)
 
     yield target.Extend('light', board=Bl602Board.BL602BOARD, app=Bl602App.LIGHT)
+    yield target.Extend('light-2000000', board=Bl602Board.BL602BOARD, app=Bl602App.LIGHT, baudrate=2000000)
```

Comment:
`2000000` is a long number to count 0s. How about naming `light-2m-baud` or similar?

Reply:
changed to 2m-uart